### PR TITLE
[DownloadBuildArtifactsV0] Normalize download path

### PR DESCRIPTION
**Pull request by vladislav-ryzhov**
**Task name**: DownloadBuildArtifactsV0

**Description**: Added path.normalize method when download path is received from the input

**Documentation changes required:** N

**Added unit tests:** N

**Checklist**:
- [X] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [X] Checked that applied changes work as expected

### CODE REVIEW

**Review by DaniilShmelev — APPROVED** [review]
LGTM, Thanks! I'll test it manually also

**Review by DenisRumyantsev — APPROVED** [review]

File: Tasks/DownloadBuildArtifactsV0/DownloadHandlers/DownloadHandlerContainerZip.ts [hunk 1]
```diff
@@ -33,12 +33,13 @@ export class DownloadHandlerContainerZip extends DownloadHandler {
      * @returns {Promise<void>} promise that will be resolved once the archive will be unpacked
     */
     private unzipContainer(unzipLocation: string): Promise<void> {
+        const normalizedUnzipLocation = path.normalize(unzipLocation);
```
Comment (anatolybolshakov) [inline Tasks/DownloadBuildArtifactsV0/DownloadHandlers/DownloadHandlerContainerZip.ts:36]: Could it make sense to normailze it earlier? Maybe in https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/DownloadBuildArtifactsV0/main.ts#L82

Comment (vladislav-ryzhov) [reply Tasks/DownloadBuildArtifactsV0/DownloadHandlers/DownloadHandlerContainerZip.ts]: @anatolybolshakov good point, thanks!

**Comment by DaniilShmelev** [comment]
Tested these changes manually, found no issues

**Review by anatolybolshakov — APPROVED** [review]
LGTM, thanks!

**Pull request merged**